### PR TITLE
Add Getting Started section to the Developer Guide

### PR DESCRIPTION
**Pull request by tmarkley**
### Description
* Add development environment setup link to Welcome section of the README.
* Remove broken Admin Responsibilities link from the README.

Signed-off-by: Tommy Markley <markleyt@amazon.com>
 
### Issues Resolved
Resolves https://github.com/opensearch-project/OpenSearch-Dashboards/issues/666
 
### Check List
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: DEVELOPER_GUIDE.md [hunk 1]
```diff
@@ -16,6 +16,32 @@ to all development within the OpenSearch Dashboards project. Please make sure to
...
+
+### Run OpenSearch Dashboards
+* Install [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md) to use the Node 10.24.1 version as it is required
+  * ```curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash```
+  * ```nvm install v10.24.1```
+* [Yarn](https://classic.yarnpkg.com/en/docs/install) is required for building and running the OpenSearch Dashboards
+  * ```npm install -g yarn```
+* Run ```yarn osd bootstrap``` in the OpenSearch Dashboards directory
+* While OpenSearch is running locally, run ```yarn start```
+* You can now navigate to ```http://localhost:5603``` where Dashboards runs by default
```
Comment (kavilla) [inline DEVELOPER_GUIDE.md:42]: really? i just navigate to 5601 directly.

**Review by kavilla — APPROVED** [review]

File: DEVELOPER_GUIDE.md (hunk 1 above)
Comment (tmarkley) [reply DEVELOPER_GUIDE.md:42]: Oh you're right, this was left over from the previous instructions.

**Comment by galangel** [comment]
I'm using [volta](https://volta.sh/) for version management, am I the only one?

**Comment by tmarkley** [comment]
@galangel We haven't adopted Volta, but it looks useful. Do you have a suggestion for incorporating it into the Getting Started instructions?

**Review by kavilla — APPROVED** [review]

**Pull request merged**